### DISCUSSION TITLE
Can I use the Terraform script for GKE with the autopilot?

### DISCUSSION THREAD

**Discussion by macurandb** [eclipse-theia/theia-cloud#242] [General]
I made some changes, but I cannot run the script with AutoPilot. Could you give me some suggestions about the changes I need to make?

**Comment by jfaltermeier** [comment]
I haven't tried this myself, but there seems to be support for autopilot: https://www.hashicorp.com/blog/terraform-adds-support-for-gke-autopilot
The GKE configuration for TheiaCloud can be found here: https://github.com/eclipsesource/theia-cloud/blob/main/terraform/modules/cluster_creation/gke/main.tf
Maybe there is just `enable_autopilot = true` missing

**Reply by macurandb** [reply]
I made this change in cluster_creation/gke/main.tf but the script failed, I will make some changes to solve the problem, and if I find the solution I will tell you. Thanks a lot
